src/SprykerEco/Zed/Payone/Business/Payment/MethodSender/PayoneGetSecurityInvoiceMethodSender.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodSender;
+
+use Generated\Shared\Transfer\PayoneGetSecurityInvoiceTransfer;
+use Generated\Shared\Transfer\PayoneStandardParameterTransfer;
+use SprykerEco\Shared\Payone\PayoneApiConstants;
+use SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface;
+use SprykerEco\Zed\Payone\Business\Api\Response\Container\AbstractResponseContainer;
+use SprykerEco\Zed\Payone\Business\Api\Response\Container\GetSecurityInvoiceResponseContainer;
+use SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReader;
+use SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface;
+
+class PayoneGetSecurityInvoiceMethodSender implements PayoneGetSecurityInvoiceMethodSenderInterface
```

Comment:
PayoneGetSecurityInvoiceReader.

Reply:
Why reader? It works as all other methods. It sends a request to payone

Reply:
++